editor/block-toolbar/index.js

```diff
@@ -78,6 +79,7 @@ class BlockToolbar extends Component {
 							<BlockInspectorButton small />
 							<BlockModeToggle uid={ block.uid } small />
 							<BlockDeleteButton uids={ [ block.uid ] } small />
+							<UnknownConverter uid={ uid } small />
```

Comment:
It seems like `block.uid` should be passed as a prop instead.

Reply:
Eslint agrees with me:
> /Users/gziolo/PhpstormProjects/vagrant-local/www/wordpress-default/public_html/wp-content/plugins/editor/block-toolbar/index.js
>  82:32  error  'uid' is not defined  no-undef

Reply:
Whoops